ppa/archive/templates/archive/digitizedwork_detail.html

```diff
@@ -42,6 +45,12 @@ <h1 class="header">{{ object.title }}</h1>
                 <td>{{ object.subtitle }}</td>
             </tr>
             {% endif %}
+            {% if object.book_journal %}
+            <tr class="book-journal">
+                <th scope="row">{% if object.item_type == "A" %}Journal{% elif object.item_type == "E" %}Book{% endif %} Title</th>
```

Comment:
same comment about the constants

I wonder if this should be a method on the class, but IDK if we'll need it anywhere else